clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java

```diff
@@ -1246,4 +1247,199 @@ public void shouldBindRequiredCronJobDockerImage() throws IOException, Interrupt
+        KubeTestUtils.loadYaml("classpath:manifests/crd_v1.yml")
+            .withValue("metadata.name", crdName)
+            .withValue("spec.scope", "Cluster")
+            .withValue("spec.group", crdGroup)
+            .asList();
+    final List<Map<String, Object>> crManifest =
+        KubeTestUtils.loadYaml("classpath:manifests/cr_v1.yml")
+            .withValue("apiVersion", String.format("%s/v1", crdGroup))
+            .withValue("metadata.name", crName)
+            .asList();
+    // ------------------------- when --------------------------
+    final List<Map<String, Object>> crdRequest =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", crdManifest)
+            .asList();
+    final List<Map<String, Object>> crRequest =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", crManifest)
+            .asList();
+    final List<String> crdDeploymets = KubeTestUtils.sendOperation(baseUrl(), crdRequest, "");
+    final List<String> crDeploymets = KubeTestUtils.sendOperation(baseUrl(), crRequest, "");
+    // ------------------------- then --------------------------
+    String crdExists = kubeCluster.execKubectl(String.format("get crd %s", crdName));
+    String crExists = kubeCluster.execKubectl(String.format("get %s %s", crdName, crName));
+    String crExistsNamespaced =
+        kubeCluster.execKubectl(String.format("-n %s get %s %s", account1Ns, crdName, crName));
+    assertTrue(
+        crdDeploymets.size() == 1
+            && crdDeploymets.get(0).equals(String.format("customResourceDefinition %s", crdName))
+            && crdExists.contains(crdName)
+            && crDeploymets.size() == 1
+            && crDeploymets.get(0).equals(String.format("CronTab.%s %s", crdGroup, crName))
+            && crExists.contains(crName)
+            && crExistsNamespaced.contains(crName));
+  }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given a deployed CRD manifest outside of spinnaker\n"
+          + "When sending credentials request\n"
+          + "Then the credentials response contains the deployed CRD\n===")
+  @Test
+  public void shouldGetDeployedCrdsCredentials() throws IOException, InterruptedException {
+    // ------------------------- given --------------------------
+    final String crdName = "crontabs.stable.example.com";
+    final Map<String, Object> crdManifest =
+        KubeTestUtils.loadYaml("classpath:manifests/crd_v1.yml")
+            .withValue("metadata.name", crdName)
+            .withValue("spec.scope", "Namespaced")
+            .asMap();
+    kubeCluster.execKubectl(" apply -f -", crdManifest);
```

Comment:
This part about deploying the CRD is not needed, because it is already hardcoded in the account configuration. This means that the test case can directly start with the `/credentials` request.

When clouddriver is fixed to show dynamic CRDs in `/credentials` then we could have a test case that deploy the CRD before checking `/credentials`